templates/hotgraphic.hbs

```diff
@@ -51,6 +51,15 @@
           {{/each}}
         </div>
       </div>
+      {{#compare _pinStyle '=' 'tiles' }}
+        <div class="hotgraphic-narrative">
+          {{#each _items}}
+          <button class="base hotgraphic-graphic-pin component-item-text-color {{#if _isVisited}}visited{{/if}} hotgraphic-graphic-pin-{{@index}} {{#if _graphic._classes}}{{_graphic._classes}}{{/if}}" data-id="item-{{@index}}" style="top:{{{_top}}}%; left:{{{_left}}}%;" role="button" aria-label="Item {{numbers @index}}. {{title}}.{{#if _isVisited}} {{../../_globals._accessibility._ariaLabels.visited}}.{{/if}}">
+            <div class="hotgraphic-graphic-pin-image component-item-color item-{{@index}}" style="background-image: url({{_graphic.src}})"></div>
+          </button>
+          {{/each}}
+        </div>
```

Comment:
Yeah, this wasn't great but since we don't have NOT equals, I couldn't find another way to do it.